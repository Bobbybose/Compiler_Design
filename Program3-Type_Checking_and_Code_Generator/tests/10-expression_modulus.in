int a = 2;
char c = 'g';
int d = a % c;