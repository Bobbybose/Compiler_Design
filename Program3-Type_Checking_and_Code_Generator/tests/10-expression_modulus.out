Code:
%t1 = 2
a = %t1
%t2 = 'g'
c = %t2
%t3 = char2int c
%t4 = a % %t3
%t5 = %t4
d = %t5

